db/migrate/20210329225033_change_data_type_for_contact_number_suppliers.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeDataTypeForContactNumberSuppliers < ActiveRecord::Migration[6.0]
+  def self.up
+    change_table :suppliers do |t|
+      t.change :contact_number, :string
+    end
+  end
+  def self.down
+    change_table :suppliers do |t|
+      t.change :contact_number, :integer
+    end
+  end
+end
```

Comment:
You can remove the keyword `.self` on defining the `up` and `down` method

Reply:
You can also use the `change_column` for this migration instead of `change_table`